include/Peer.hpp

```diff
@@ -21,6 +24,10 @@ class Peer
 		void				clearMessage(void);
 		bool				hasCompleteMessage(void) const;
 		const std::string	&getMessage(void) const;
+		bool				isRegistered(void) const;
+		void				registration(const std::string &user, const std::string &mode, const std::string &realname);
```

Comment:
`register` is a better name according to me. What do you think @theovgl ?

Reply:
c'est noté.